### ISSUE TITLE
about implemention of GST

### ISSUE THREAD

**Issue by haoyu0408** [chijames/GST#1]
Thanks for your nice work! It is really interesting! But I have one question why we should subtract the mean of logits (logits = logits - logits.mean(dim=-1, keepdim=True)). Would you like to help me?

**Comment by chijames** [comment]
Hi,

Thanks for your interest in our work. This is just for numerical stability as Softmax output does not change if we add/subtract the input logits by a constant.
